NEWS.md

```diff
@@ -1,4 +1,7 @@
 # ggplot2 3.1.0.9000
+  
+* `scale_color_continuous()` now points at `scale_colour_continuos()` so that it 
+  will handle `type` = "viridis" as the documentation states (@hlendway, #3079).
```

Comment:
The closing backtick should be after `"viridis"`, as in `` `type = "viridis"` ``.